Client/Frontend/Settings/Brave News/BraveNewsDebugSettingsView.swift

```diff
@@ -0,0 +1,214 @@
+// Copyright 2020 The Brave Authors. All rights reserved.
```

Comment:
Update year

Reply:
This file was actually renamed, but git for some reason picked it up as a new file lol. I can update it anyways though

Reply:
I've added an assert above this to check for non-empty supported langs as that being empty is never supposed to happen anyways

Reply:
thanks, works for me